### PR TITLE
BAH-2020 Create Lab-lite Smoke Test with current functionalities

### PR DESCRIPTION
**Pull request by gokultw**
Modified Lab-lite Test step Implementations & concepts

### CODE REVIEW

File: tests/util/taikoHelper.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ async function repeatUntilNotVisible(element) {
     var isFound = true;
     do {
         try {
-            if (await element.exists(0, 0)) {
```
Comment (gsluthra) [inline tests/util/taikoHelper.js:23]: This method looks like it was used in other places too. Why was this change needed?

**Comment by gsluthra** [comment]
To make the lab report look a little bit realistic, can we use an image/pdf from here:
https://bahmni.atlassian.net/wiki/spaces/BAH/pages/2870280206/OCR+in+Bahmni

**Comment by gokultw** [comment]
> To make the lab report look a little bit realistic, can we use an image/pdf from here: https://bahmni.atlassian.net/wiki/spaces/BAH/pages/2870280206/OCR+in+Bahmni

Updated the lab report

File: tests/lab/labLite.js [hunk 2]
```diff
@@ -75,13 +81,17 @@ step("Select Doctor in side panel", async function () {
...
 });
 
-step("Verify the uploaded report", async function() {
-	click(link("labReport1.jpg"));
-    assert.ok(await $("//h3[text()='1kbhsg.jpeg']//..//..//img").isVisible());
-    button("close").click();
+step("Verify the uploaded report", async function () {
+    await click("labReport1.jpg");
+    await highlight($("//DIV[contains(@class,'is-visible')]//IMG/../..//h3[text()='labReport1.jpg']"))
+    await highlight($("//DIV[contains(@class,'is-visible')]//IMG"))
```
[the remark this answers is not in the document]
Comment (gsluthra) [reply tests/lab/labLite.js:94]: Maybe this is another use-case to inform Taico team about, and see if they have any opinion. For now I will mark this as resolved. thanks!

**Review by gsluthra — APPROVED** [review]
